hplsql/src/test/results/local/drop_package.out.txt

```diff
@@ -0,0 +1,16 @@
+Ln:1 CREATE PACKAGE
+Ln:7 CREATE PACKAGE BODY
+Ln:2 DECLARE count INT = 0
+Ln:13 EXEC PACKAGE PROCEDURE COUNTER.INC
+Ln:13 SET PARAM i = 10
+Ln:8 SET count = 10
+Ln:15 PRINT
+EXEC PACKAGE FUNCTION COUNTER.CURRENT
+Ln:7 RETURN
+10
+Ln:17 DROP
+Ln:17 COUNTER DROPPED
+Ln:19 PRINT
+null
```

Comment:
why didn't this resulted in an exception? I think asking for an undefined variable inside an unknown package should be a hard error!

this behaviour may help introduce bugs (and keep them hidden) into the hplsql codes users write.

Reply:
Yes, I agree, and the same is true for calling an undefined function (=null) accessing to an undefined global variable (=null) or adding an undefined variable to a number (=null).

There is a jira for these kind of issues (HIVE-24315) and as I remember I already fixed all of these in the (yet unpublished) patch.